### PR TITLE
[New Hack] 058-Power Platform Basics

### PR DESCRIPTION
**Pull request by tadthompson**

### CODE REVIEW

File: 058-PowerPlatformBasics/Coach/Solution-01.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Challenge 01 - Challenge Setup - Coach's Guide 
+
+[< Previous Solution](./Solution-00.md) - **[Home](./README.md)** - [Next Solution >](./Solution-02.md)
```
Comment (jordanbean-msft) [inline 058-PowerPlatformBasics/Coach/Solution-01.md:3]: this is a navigation link to "Solution-00", which is typically prerequisites. your WTH doesn't have a solution 00, you put the setup in challenge/solution 01, either re-number all of your solutions & challenges (lots of work :)) or remove this navigation link to non-existent files

something like:
`**[Home](./README.md)** - [Next Solution >](./Solution-02.md)`

File: 058-PowerPlatformBasics/Coach/Solution-01.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Notes & Guidance
+
+Environment setup guidence
+- Ensure they create a Power Platform trial - with a database
+- Ensure users use in-private/incognito or create a new profile in their browser (do not use the default browser to create a new environment as it will conflict with their work O365 login)
+- Do not use your corporate license/environment for the hack
+- Create a unique tenant name such as: ppwth2022TeamXX
+
+- If Microsoft employees and Partners have access to CDX, we reccommend they stil follow the trial setup process
```
Comment (jordanbean-msft) [inline 058-PowerPlatformBasics/Coach/Solution-01.md:13]: typos:

"recommend"

"still"

File: 058-PowerPlatformBasics/Coach/Solution-02.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Challenge 02 - Handling Incoming Orders- Coach's Guide 
+
+[< Previous Solution](./Solution-01.md) - **[Home](./README.md)** - [Next Solution >](./Solution-03.md)
+
+## Notes & Guidance
+
+- Prebuilt Invoice processing Model can extract majority of data on given Order Forms.
+- Using the extracted Customer data on the form to find existing customer in the system, if not found, create a new customer record. 
+- The outcome of AI Builder Model are texts. We need to convert data to appropriate type when creating a new record in Order and Order Line Item Table.
```
Comment (jordanbean-msft) [inline 058-PowerPlatformBasics/Coach/Solution-02.md:9]: "is text"

"to the appropriate"

File: 058-PowerPlatformBasics/Coach/Solution-02.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+- In case, AI Builder Model cannot extract data from the form, design a Fall Back approach to log the failed run and send email alert/post on Team to Admin.
```
Comment (jordanbean-msft) [inline 058-PowerPlatformBasics/Coach/Solution-02.md:10]: "Teams"

File: 058-PowerPlatformBasics/Coach/Solution-04.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Challenge 04 - Automation - Approval Process - Coach's Guide 
+
+[< Previous Solution](./Solution-03.md) - **[Home](./README.md)** - [Next Solution >](./Solution-05.md)
+
+## Notes & Guidance
+
+This is the only section you need to include.
```
Comment (jordanbean-msft) [inline 058-PowerPlatformBasics/Coach/Solution-04.md:7]: remove boilerplate text

"This is the only section you need to include.

Use general non-bulleted text for the beginning of a solution area for this challenge"

**Review by jordanbean-msft — CHANGES_REQUESTED** [review]

**Review by jordanbean-msft — APPROVED** [review]

**Review by jordanbean-msft — APPROVED** [review]

**Review by jrzyshr — APPROVED** [review]
Thanks team for our first Power Platform What The Hack!!!!!

File: 058-PowerPlatformBasics/Student/Challenge-01.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+### Choose the platform and build environment
+  
+- Answer the question:  What platform will you choose and why?
+- Form a team and assign roles.  Recommended:  Choose a method of collaboration (e.g. OneNote, Teams, etc.)
+- Provision requirements for Power Platform.
+- To begin this challenge, Tailspin Toys would like your team to create a new solution in your team's development environment. This new solution will serve as a container for the components that your team will be assigned in later challenges. In accordance with application lifecycle management (ALM) best practices, all future work should be created from this solution
+
+**Bonus Challenge:** Have the team present to the Tailspin Toys board their platform of choice and why.  Include milestones if confident.
+
+**NOTE:** This challenge will require licenses to complete the challenge using trials
```
[the remark this answers is not in the document]
Comment (jrzyshr) [reply 058-PowerPlatformBasics/Student/Challenge-01.md]: Fixed with new PDF in Student/Resources

**Pull request merged**